Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -161,6 +173,11 @@ private void OnIsHotEvent(EntityUid uid, FlammableComponent flammable, IsHotEven
 
         private void OnTileFireEvent(EntityUid uid, FlammableComponent flammable, ref TileFireEvent args)
         {
+            // Don't set yourself on fire even more
+            // Limiting to specific tags for now since self extinguishing doesn't work if this is on
+            if (flammable.OnFire || !_tags.HasTag(uid, "Paper"))
+                return;
+
```

Comment:
The *whole point* of this event is that the entire tile is on fire and things on it are caught in the flames... Remove this.